hack/test/manifests/cis-kube-bench-master.yaml

```diff
@@ -17,8 +17,7 @@ spec:
         image: aquasec/kube-bench:latest
         args:
         - master
-        - --json
-        - --version=1.11
+        - --version=1.13
```

Comment:
Is 1.13 the latest they support?

Reply:
Yep. https://github.com/aquasecurity/kube-bench#cis-kubernetes-benchmark-support shows 1.13 as the latest and it's the latest also specified in their cfg/ directory in master branch.